### DISCUSSION TITLE
GetPostingTransactionSummaryRequest not found

### DISCUSSION THREAD

**Discussion by fruitizz** [netsuitephp/netsuite-php#150] [General]
I am trying to use the `GetPostingTransactionSummaryRequest` and I am getting the error below.
```ruby
$service = app(NetSuiteService::class);
$request = new GetPostingTransactionSummaryRequest();
```
The challenge is that the `GetPostingTransactionSummaryRequest` is in the vendor NetSuite folder but not able to call it. Is there anything I may be missing? Thanks.

![Screenshot 2023-05-04 at 14 40 55](https://user-images.githubusercontent.com/10519310/236224751-218a6191-e9ba-48a2-92eb-2f3b632a67b6.png)

**Comment by jrebs** [comment]
There appears to be something wrong with your application configuration in some way that it's not autoloading your classes or something. There's not really enough information here to diagnose, but I can confirm that in my own laravel application utilizing this package, I'm able to instantiate the class just fine.

![image](https://user-images.githubusercontent.com/4203789/236270775-44b62b6f-6c0a-496e-94ca-d6c33418b02a.png)
